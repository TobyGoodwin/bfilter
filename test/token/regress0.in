Received: from uspmta169080.emsmtp.com [212.69.169.80] by mx.flare.email
    with SMTP; 29 Jul 2014 11:10:25 -0000
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; s=key2; d=mailsys.buyassociation.co.uk;
 h=To:Subject:From:MIME-Version:List-Id:List-Unsubscribe:Content-Type:Message-ID:Date; i=reply@mailsys.buyassociation.co.uk;
 bh=leblXnkcEwuDD6c6pK133HmW3fE=;
 b=KGr8yX/YnNxozQpOK4PWfvh0Se3Fxea/VHByCGWj9S0EC/czaivDLZhS48ee5SmB7yy5OoEcZLh+
   usvmcvVZpjfPymieyAEv2FEkL3EO0DfM6IoFvqLNHOgntjK2WVP3afjCOhoMgLRXDQFS0PLk0fxS
   B24xKZ35A7rtr3+4n5I=
DomainKey-Signature: a=rsa-sha1; c=nofws; q=dns; s=key2; d=mailsys.buyassociation.co.uk;
 b=a85s0/EBGyEuZO3cHFiovC4XKnh6eTuDhsqN4DxmiwjDQn1/5lfUAnSsLdbEJecVL51DOQXA+bnw
   k2LuWLHhA7CdRYYGg8fRMmBvrIHVXfdytpGK4q0SnbyW+6riDp8tMh7n/iQubJUXQnF0suYvKFoS
   etONqFtnv9OUPoTLO0k=;
Received: from (10.200.200.159) by uspmta169080.emsmtp.com id hqu0i016necl for <info@libra-aries-books.co.uk>; Tue, 29 Jul 2014 13:10:21 +0200 (envelope-from <wwwrun@xpressus.emsmtp.com>)
To: info@libra-aries-books.co.uk
Subject: Interested in investing in UK =?UTF-8?Q?Property=3F?=
X-Mailer: class SMTPMail
From: "BuyAssociation" <reply@mailsys.buyassociation.co.uk>
MIME-Version: 1.0
List-Id: 198797450 <BuyAssociation>
X-EMarSys-Identify: 198797450_835960_22474
X-EMarSys-Environment: xpress_us
X-CSA-Complaints: whitelist-complaints@eco.de
List-Unsubscribe: <http://links.buyassociation.co.uk/u/lu.php?lu=V,rzr:x2Kd:bvb:TdLeZ6eVnG>
Content-Type: multipart/alternative;
 boundary="--=_---NextPart--=_-I9iGNUvPzu"
Message-ID: <0.0.AE.F01.1CFAB1DB009B75E.0@uspmta169080.emsmtp.com>
Date: Tue, 29 Jul 2014 13:10:21 +0200


----=_---NextPart--=_-I9iGNUvPzu
Content-Type: text/plain; charset=utf-8
Content-transfer-encoding: quoted-printable

To view the graphic version (HTML) of this e-mail, click on the
following link or copy it into your browser:
http://links.buyassociation.co.uk/u/gm.php?prm=3DTdLeZ6eVnG_198797450_83596=
0_22474

UK HMO Investments Contracted=20
10% Net Returns for 3-5 Years
UK HMO Investments Contracted
=20
http://links.buyassociation.co.uk/u/nrd.php?p=3DTdLeZ6eVnG_22474_835960_1_2=
0
 =09
Property #1
High Street,
Barnsley
=C2=A355,000
split apartment
=20

Property #2
Avenue Road,=20
Rotherham=20
=C2=A3104,950
4 bed
=20
 =09
Property #3
Rawmarsh,
Rotherham
=C2=A3179,950
7 bed
=20
=20
Fully managed high yielding investment over 200 properties under
management
=20
=20
Investment Highlights=20
 =09
=20
=E2=9C=93=09RICS valuation
=E2=9C=93=09Full title from day 1
=E2=9C=93=09Professionally planned & refurbished
=E2=9C=93=09FREE furniture throughout
=E2=9C=93=09Fully tenanted & fully managed
=E2=9C=93=09HMO compliancy and licenses
=E2=9C=93=09Contracted developer Buy Back Option
=E2=9C=93=09Hands off investment
=20

http://links.buyassociation.co.uk/u/nrd.php?p=3DTdLeZ6eVnG_22474_835960_1_2=
1

You are receiving emails from us as your email is part of a
business-to-business email database which was created from email
addresses available in the public domain. Should you wish not to receive
any further communications, please use the unsubscribe link on the
email.

If you want to unsubscribe from our mailing list, unsubscribe
http://links.buyassociation.co.uk/u/un.php?par=3DTdLeZ6eVnG_835960_22474_$s=
id$
----=_---NextPart--=_-I9iGNUvPzu
Content-Type: text/html; charset=utf-8
Content-transfer-encoding: quoted-printable

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.=
w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns=3D"http://www.w3.org/1999/xhtml">
<head>=09<meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3D=
UTF-8" />=09<title>IA Property</title>=09<style type=3D"text/css">=09=09=09=
#outlook a {padding:0;} =09body{width:100% !important; -webkit-text-size-ad=
just:100%; -ms-text-size-adjust:100%; margin:0; padding:0;}=09=09.ExternalC=
lass {width:100%;} =09.ExternalClass, .ExternalClass p, .ExternalClass span=
, .ExternalClass font, .ExternalClass td, .ExternalClass div {line-height: =
100%;}=09=09#backgroundTable {margin:0; padding:0; width:100% !important; l=
ine-height: 100% !important;}=09=09=09img {outline:none; text-decoration:no=
ne; -ms-interpolation-mode: bicubic;}=09img {display:block};=09=09table td =
{border-collapse: collapse;}=09=09table { border-collapse:collapse; mso-tab=
le-lspace:0pt; mso-table-rspace:0pt; }=09</style>
</head>
<body><table width=3D"100%" cellpadding=3D"0" cellspacing=3D"0" border=3D"0=
"><tr><td align=3D"center"><font face=3D"Arial, Helvetica" size=3D"1" style=
=3D"font-size:9px;color:#6d6d6d">If this e-mail is not displayed properly, =
click <a href=3D"http://links.buyassociation.co.uk/u/gm.php?prm=3DTdLeZ6eVn=
G_198797450_835960_22474" target=3D"_blank"><font face=3D"Arial, Helvetica"=
 size=3D"1" style=3D"font-size:9px;color:#6d6d6d; text-decoration: none;"> =
here.</font></a><br></font></td></tr></table>=09<table class=3D"backgroundT=
able" width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0" heigh=
t=3D"100%">=09<tr>=09<td valign=3D"top" align=3D"center">=09<table width=3D=
"648" border=3D"0" cellpadding=3D"0" cellspacing=3D"0" style=3D"border: 1px=
 solid #D1D1D1;">=09<!-- HERO -->=09<tr bgcolor=3D"#9bcbeb">=09<td colspan=
=3D"3">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td=
 colspan=3D"3" align=3D"center" bgcolor=3D"#9bcbeb">=09<font size=3D"5" sty=
le=3D"font-family:Arial;font-size:36px;color:#ffffff;">=09<strong>=09UK HMO=
 Investments Contracted <br/>=0910% Net Returns for 3-5 Years=09</strong>=
=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=3D"3">=09<a href=3D"http://=
links.buyassociation.co.uk/u/nrd.php?p=3DTdLeZ6eVnG_22474_835960_1_15" targ=
et=3D"_blank"><img src=3D"http://6a439d83223879fb42ed-fc289d1a1e5f3b39b56dc=
847c1628551.r97.cf3.rackcdn.com/fixedfee/hero.jpg" width=3D"648" height=3D"=
278" alt=3D"UK HMO Investments Contracted"></a>=09</td>=09</tr>=09<!-- TOP =
PARAGRAPH -->=09<tr>=09<td colspan=3D"3">=09<table width=3D"100%" border=3D=
"0" cellpadding=3D"0" cellspacing=3D"0" bgcolor=3D"#ebdfd1">=09<tr>=09<td c=
olspan=3D"3">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=
=09<td colspan=3D"1" width=3D"20">=09&nbsp;=09</td>=09<td colspan=3D"1" wid=
th=3D"538">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0" cellspac=
ing=3D"0">=09<tr>=09<td>=09<table width=3D"100%" border=3D"0" cellpadding=
=3D"0" cellspacing=3D"0">=09<tr>=09<td colspan=3D"1" align=3D"center" width=
=3D"187">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacin=
g=3D"0">=09<tr>=09<td>=09<a href=3D"http://links.buyassociation.co.uk/u/nrd=
.php?p=3DTdLeZ6eVnG_22474_835960_1_16" target=3D"_blank"><img src=3D"http:/=
/6a439d83223879fb42ed-fc289d1a1e5f3b39b56dc847c1628551.r97.cf3.rackcdn.com/=
fixedfee/image1.jpg" height=3D"141" width=3D"188" alt=3D"Property #1"></a>=
=09</td>=09</tr>=09<tr>=09<td align=3D"center" bgcolor=3D"#643410">=09<tabl=
e width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=
=09<td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09<=
/tr>=09<tr>=09<td align=3D"center">=09<font size=3D"3" style=3D"font-family=
:Arial;color:#ffffff;font-size:16px;">=09<strong>=09High Street,<br/>=09Bar=
nsley<br/>=09&pound;55,000=09</strong>=09</font>=09</td>=09</tr>=09<tr>=09<=
td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=
=09</table>=09</td>=09</tr>=09<tr>=09<td align=3D"center" bgcolor=3D"#ce6f1=
f">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0=
">=09<tr>=09<td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09=
</td>=09</tr>=09<tr>=09<td align=3D"center">=09<font size=3D"3" style=3D"fo=
nt-family:Arial;color:#ffffff;font-size:16px;">=09<strong>=09split apartmen=
t=09</strong>=09</font>=09</td>=09</tr>=09<tr>=09<td align=3D"center">=09<f=
ont size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09</table>=09</td>=09</t=
r>=09</table>=09</td>=09<td colspan=3D"1" width=3D"20">=09&nbsp;=09</td>=09=
<td colspan=3D"1" align=3D"center" width=3D"187">=09<table width=3D"100%" b=
order=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=09<td>=09<a href=3D=
"http://links.buyassociation.co.uk/u/nrd.php?p=3DTdLeZ6eVnG_22474_835960_1_=
17" target=3D"_blank"><img src=3D"http://6a439d83223879fb42ed-fc289d1a1e5f3=
b39b56dc847c1628551.r97.cf3.rackcdn.com/fixedfee/image2.jpg" height=3D"141"=
 width=3D"188" alt=3D"Property #2"></a>=09</td>=09</tr>=09<tr>=09<td align=
=3D"center" bgcolor=3D"#643410">=09<table width=3D"100%" border=3D"0" cellp=
adding=3D"0" cellspacing=3D"0">=09<tr>=09<td align=3D"center">=09<font size=
=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td align=3D"center">=
=09<font size=3D"3" style=3D"font-family:Arial;color:#ffffff;font-size:16px=
;">=09<strong>=09Avenue Road, <br/>=09Rotherham <br/>=09&pound;104,950=09</=
strong>=09</font>=09</td>=09</tr>=09<tr>=09<td align=3D"center">=09<font si=
ze=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09</table>=09</td>=09</tr>=09<=
tr>=09<td align=3D"center" bgcolor=3D"#ce6f1f">=09<table width=3D"100%" bor=
der=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=09<td align=3D"center=
">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td alig=
n=3D"center">=09<font size=3D"3" style=3D"font-family:Arial;color:#ffffff;f=
ont-size:16px;">=09<strong>=094 bed=09</strong>=09</font>=09</td>=09</tr>=
=09<tr>=09<td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09</=
td>=09</tr>=09</table>=09</td>=09</tr>=09</table>=09</td>=09<td colspan=3D"=
1" width=3D"20">=09&nbsp;=09</td>=09<td colspan=3D"1" align=3D"center" widt=
h=3D"187">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0" cellspaci=
ng=3D"0">=09<tr>=09<td>=09<a href=3D"http://links.buyassociation.co.uk/u/nr=
d.php?p=3DTdLeZ6eVnG_22474_835960_1_18" target=3D"_blank"><img src=3D"http:=
//6a439d83223879fb42ed-fc289d1a1e5f3b39b56dc847c1628551.r97.cf3.rackcdn.com=
/fixedfee/image3.jpg" height=3D"141" width=3D"189" alt=3D"Property #3"></a>=
=09</td>=09</tr>=09<tr>=09<td align=3D"center" bgcolor=3D"#643410">=09<tabl=
e width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=
=09<td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09<=
/tr>=09<tr>=09<td align=3D"center">=09<font size=3D"3" style=3D"font-family=
:Arial;color:#ffffff;font-size:16px;">=09<strong>=09Rawmarsh,<br/>=09Rother=
ham<br/>=09&pound;179,950=09</strong>=09</font>=09</td>=09</tr>=09<tr>=09<t=
d align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=
=09</table>=09</td>=09</tr>=09<tr>=09<td align=3D"center" bgcolor=3D"#ce6f1=
f">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0=
">=09<tr>=09<td align=3D"center">=09<font size=3D"1">=09&nbsp;=09</font>=09=
</td>=09</tr>=09<tr>=09<td align=3D"center">=09<font size=3D"3" style=3D"fo=
nt-family:Arial;color:#ffffff;font-size:16px;">=09<strong>=097 bed=09</stro=
ng>=09</font>=09</td>=09</tr>=09<tr>=09<td align=3D"center">=09<font size=
=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09</table>=09</td>=09</tr>=09</t=
able>=09</td>=09</tr>=09</table>=09</td>=09</tr>=09<tr>=09<td>=09<font size=
=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td align=3D"center">=
=09<font size=3D"5" style=3D"font-family:Arial;color:#000000;font-size:28px=
;">=09<strong>=09Fully managed high yielding investment<br/>=09over 200 pro=
perties under management=09</strong>=09</font>=09</td>=09</tr>=09</table>=
=09</td>=09<td colspan=3D"1" width=3D"20">=09&nbsp;=09</td>=09</tr>=09<tr>=
=09<td colspan=3D"3" >=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</t=
r>=09</table>=09</td>=09</tr>=09<tr>=09<td colspan=3D"3" bgcolor=3D"#ebdfd1=
">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<!-- BOTTOM PAR=
AGRAPH -->=09<tr>=09<td colspan=3D"3" bgcolor=3D"#bb9566">=09<font size=3D"=
1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=3D"3">=09<table=
 width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0" bgcolor=3D=
"#bb9566">=09<tr>=09<td colspan=3D"3">=09<table width=3D"100%" border=3D"0"=
 cellpadding=3D"0" cellspacing=3D"0">=09<tr>=09<td colspan=3D"3">=09<table =
width=3D"100%" border=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=09<=
td colspan=3D"5" align=3D"center">=09<font size=3D"5" style=3D"font-family:=
Arial;color:#000000;font-size:26px;">=09<strong>=09Investment Highlights=09=
</strong>=09</font><br/>=09<hr width=3D"90%">=09</td>=09</tr>=09<tr>=09<td =
colspan=3D"1" width=3D"20">=09&nbsp;=09</td>=09<td colspan=3D"1" align=3D"c=
enter" width=3D"179">=09<table width=3D"100%" border=3D"0" cellpadding=3D"0=
" cellspacing=3D"0">=09<tr>=09<td colspan=3D"3">=09<font size=3D"1">=09&nbs=
p;=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=3D"1" width=3D"30" align=
=3D"center" valign=3D"top">=09<font face=3D"arial" size=3D"6" color=3D"#333=
333" style=3D"font-size: 24px;">=09<strong>&#10003;</strong>=09</font>=09</=
td>=09<td colspan=3D"2" width=3D"279">=09<font face=3D"arial" size=3D"4" co=
lor=3D"#333333" style=3D"font-size: 18px;">=09RICS valuation=09</font>=09</=
td>=09</tr>=09<tr>=09<td colspan=3D"1" width=3D"30" align=3D"center" valign=
=3D"top">=09<font face=3D"arial" size=3D"6" color=3D"#333333" style=3D"font=
-size: 24px;">=09<strong>&#10003;</strong>=09</font>=09</td>=09<td colspan=
=3D"2" width=3D"279">=09<font face=3D"arial" size=3D"4" color=3D"#333333" s=
tyle=3D"font-size: 18px;">=09Full title from day 1=09</font>=09</td>=09</tr=
>=09<tr>=09<td colspan=3D"1" width=3D"30" align=3D"center" valign=3D"top">=
=09<font face=3D"arial" size=3D"6" color=3D"#333333" style=3D"font-size: 24=
px;">=09<strong>&#10003;</strong>=09</font>=09</td>=09<td colspan=3D"2" wid=
th=3D"279">=09<font face=3D"arial" size=3D"4" color=3D"#333333" style=3D"fo=
nt-size: 18px;">=09Professionally planned & refurbished=09</font>=09</td>=
=09</tr>=09<tr>=09<td colspan=3D"1" width=3D"30" align=3D"center" valign=3D=
"top">=09<font face=3D"arial" size=3D"6" color=3D"#333333" style=3D"font-si=
ze: 24px;">=09<strong>&#10003;</strong>=09</font>=09</td>=09<td colspan=3D"=
2" width=3D"279">=09<font face=3D"arial" size=3D"4" color=3D"#333333" style=
=3D"font-size: 18px;">=09FREE furniture throughout=09</font>=09</td>=09</tr=
>=09</table>=09</td>=09<td colspan=3D"1" width=3D"10">=09&nbsp;=09</td>=09<=
td colspan=3D"1" align=3D"center" width=3D"179">=09<table width=3D"100%" bo=
rder=3D"0" cellpadding=3D"0" cellspacing=3D"0">=09<tr>=09<td colspan=3D"3">=
=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr>=09<tr>=09<td colspa=
n=3D"1" width=3D"30" align=3D"center" valign=3D"top">=09<font face=3D"arial=
" size=3D"6" color=3D"#333333" style=3D"font-size: 24px;">=09<strong>&#1000=
3;</strong>=09</font>=09</td>=09<td colspan=3D"2" width=3D"279">=09<font fa=
ce=3D"arial" size=3D"4" color=3D"#333333" style=3D"font-size: 18px;">=09Ful=
ly tenanted & fully managed=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=
=3D"1" width=3D"30" align=3D"center" valign=3D"top">=09<font face=3D"arial"=
 size=3D"6" color=3D"#333333" style=3D"font-size: 24px;">=09<strong>&#10003=
;</strong>=09</font>=09</td>=09<td colspan=3D"2" width=3D"279">=09<font fac=
e=3D"arial" size=3D"4" color=3D"#333333" style=3D"font-size: 18px;">=09HMO =
compliancy and licenses=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=3D"1=
" width=3D"30" align=3D"center" valign=3D"top">=09<font face=3D"arial" size=
=3D"6" color=3D"#333333" style=3D"font-size: 24px;">=09<strong>&#10003;</st=
rong>=09</font>=09</td>=09<td colspan=3D"2" width=3D"279">=09<font face=3D"=
arial" size=3D"4" color=3D"#333333" style=3D"font-size: 18px;">=09Contracte=
d developer Buy Back Option=09</font>=09</td>=09</tr>=09<tr>=09<td colspan=
=3D"1" width=3D"30" align=3D"center" valign=3D"top">=09<font face=3D"arial"=
 size=3D"6" color=3D"#333333" style=3D"font-size: 24px;">=09<strong>&#10003=
;</strong>=09</font>=09</td>=09<td colspan=3D"2" width=3D"279">=09<font fac=
e=3D"arial" size=3D"4" color=3D"#333333" style=3D"font-size: 18px;">=09Hand=
s off investment=09</font>=09</td>=09</tr>=09</table>=09</td>=09<td colspan=
=3D"1" width=3D"20">=09&nbsp;=09</td>=09</tr>=09</table>=09</td>=09</tr>=09=
</table>=09</td>=09</tr>=09</table>=09</td>=09</tr>=09<tr>=09<td colspan=3D=
"3" bgcolor=3D"#bb9566">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09<=
/tr>=09<!-- FOOTER -->=09<tr>=09<td colspan=3D"3">=09<a href=3D"http://link=
s.buyassociation.co.uk/u/nrd.php?p=3DTdLeZ6eVnG_22474_835960_1_19" target=
=3D"_blank"><img src=3D"http://6a439d83223879fb42ed-fc289d1a1e5f3b39b56dc84=
7c1628551.r97.cf3.rackcdn.com/fixedfee/footer2.jpg" width=3D"648" height=3D=
"268" alt=3D"IA Property, Click here for more."></a>=09</td>=09</tr>=09<tr>=
=09<td colspan=3D"3">=09<font size=3D"1">=09&nbsp;=09</font>=09</td>=09</tr=
>=09</table>=09</td>=09</tr>=09</table>
<html><head></head><body><table width=3D"100%" cellpadding=3D"0" cellspacin=
g=3D"0" border=3D"0"><tr><td align=3D"center"><font face=3D"Arial, Helvetic=
a" size=3D"1" style=3D"font-size:9px;color:#6d6d6d">If this e-mail is not d=
isplayed properly, click <a href=3D"http://links.buyassociation.co.uk/u/gm.=
php?prm=3DTdLeZ6eVnG_198797450_835960_22474" target=3D"_blank"><font face=
=3D"Arial, Helvetica" size=3D"1" style=3D"font-size:9px;color:#6d6d6d; text=
-decoration: none;"> here.</font></a><br></font></td></tr></table><table> <=
tr>=09<td width=3D"10"> </td>=09<td align=3D"center"><font size=3D"1" face=
=3D"arial" color=3D"#999999">You are receiving emails from us as your email=
 is part of a business-to-business email database which was created from em=
ail addresses available in the public domain. Should you wish not to receiv=
e any further communications, please use the unsubscribe link on the email.=
<br /></font></td>=09<td width=3D"10"> </td>=09</tr> </table>
<table width=3D"100%" cellpadding=3D"0" cellspacing=3D"0" border=3D"0"><tr>=
<td align=3D"center"><font face=3D"Arial, Helvetica" size=3D"1" style=3D"fo=
nt-size:9px;color:#6d6d6d"><br>If you want to unsubscribe from our mailing =
list, unsubscribe <a href=3D"http://links.buyassociation.co.uk/u/un.php?par=
=3DTdLeZ6eVnG_835960_22474_$sid$" target=3D"_blank"><font face=3D"Arial, He=
lvetica" size=3D"1" style=3D"font-size:9px;color:#6d6d6d; text-decoration: =
none;"> here.</font></a></td></tr></table><img src=3D"http://links.buyassoc=
iation.co.uk/img/TdLeZ6eVnG_198797450_835960_22474.gif" height=3D2 width=3D=
2 border=3D0>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<tr>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<colgroup>
<col width=3D"10%">
<col width=3D"80%">
<col width=3D"10%">
</colgroup>
<tr>
<td>&nbsp;</td>
<td>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<tr>
<td width=3D"100%" align=3D"center"><font face=3D"Arial, Helvetica" size=3D=
"1" style=3D"font-size:11px;color:#666666"></font></td>
</tr>
</table>
</td>
<td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td>&nbsp;</td>
</tr>
</table>
</body></html>
<table width=3D"100%" cellpadding=3D"0" cellspacing=3D"0" border=3D"0"><tr>=
<td align=3D"center"><font face=3D"Arial, Helvetica" size=3D"1" style=3D"fo=
nt-size:9px;color:#6d6d6d"><br>If you want to unsubscribe from our mailing =
list, unsubscribe <a href=3D"http://links.buyassociation.co.uk/u/un.php?par=
=3DTdLeZ6eVnG_835960_22474_$sid$" target=3D"_blank"><font face=3D"Arial, He=
lvetica" size=3D"1" style=3D"font-size:9px;color:#6d6d6d; text-decoration: =
none;"> here.</font></a></td></tr></table><img src=3D"http://links.buyassoc=
iation.co.uk/img/TdLeZ6eVnG_198797450_835960_22474.gif" height=3D2 width=3D=
2 border=3D0>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<tr>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<colgroup>
<col width=3D"10%">
<col width=3D"80%">
<col width=3D"10%">
</colgroup>
<tr>
<td>&nbsp;</td>
<td>
<table width=3D"100%" cellspacing=3D"0" cellpadding=3D"0" border=3D"0">
<tr>
<td width=3D"100%" align=3D"center"><font face=3D"Arial, Helvetica" size=3D=
"1" style=3D"font-size:11px;color:#666666"></font></td>
</tr>
</table>
</td>
<td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td>&nbsp;</td>
</tr>
</table>
</body>
</html>
----=_---NextPart--=_-I9iGNUvPzu--
