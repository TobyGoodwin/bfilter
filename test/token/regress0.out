info@libra-aries-books.co.uk
Interested
in
investing
in
UK
Property
BuyAssociation
-I9iGNUvPzu
Content-Type
text/plain
charset
utf-8
Content-transfer-encoding
quoted-printable
To
view
the
graphic
version
HTML
of
this
e-mail
click
on
the
following
link
or
copy
it
into
your
browser
http
prm
TdLeZ6eVnG
UK
HMO
Investments
Contracted
Net
Returns
for
3-5
Years
UK
HMO
Investments
Contracted
http
TdLeZ6eVnG
Property
High
Street
Barnsley
£55
split
apartment
Property
Avenue
Road
Rotherham
£104
bed
Property
Rawmarsh
Rotherham
£179
bed
Fully
managed
high
yielding
investment
over
properties
under
management
Investment
Highlights
✓
RICS
valuation
✓
Full
title
from
day
✓
Professionally
planned
refurbished
✓
FREE
furniture
throughout
✓
Fully
tenanted
fully
managed
✓
HMO
compliancy
and
licenses
✓
Contracted
developer
Buy
Back
Option
✓
Hands
off
investment
http
TdLeZ6eVnG
You
are
receiving
emails
from
us
as
your
email
is
part
of
business-to-business
email
database
which
was
created
from
email
addresses
available
in
the
public
domain
Should
you
wish
not
to
receive
any
further
communications
please
use
the
unsubscribe
link
on
the
email
If
you
want
to
unsubscribe
from
our
mailing
list
unsubscribe
http
par
TdLeZ6eVnG
sid
-I9iGNUvPzu
Content-Type
text/html
charset
utf-8
Content-transfer-encoding
quoted-printable
IA
Property
outlook
padding
body
width
important
-webkit-text-size-adjust
-ms-text-size-adjust
margin
padding
ExternalClass
width
ExternalClass
ExternalClass
ExternalClass
span
ExternalClass
font
ExternalClass
td
ExternalClass
div
line-height
backgroundTable
margin
padding
width
important
line-height
important
img
outline
none
text-decoration
none
-ms-interpolation-mode
bicubic
img
display
block
table
td
border-collapse
collapse
table
border-collapse
collapse
mso-table-lspace
0pt
mso-table-rspace
0pt
If
this
e-mail
is
not
displayed
properly
click
here
nbsp
UK
HMO
Investments
Contracted
Net
Returns
for
3-5
Years
nbsp
nbsp
nbsp
High
Street
Barnsley
pound
nbsp
nbsp
split
apartment
nbsp
nbsp
nbsp
Avenue
Road
Rotherham
pound
nbsp
nbsp
bed
nbsp
nbsp
nbsp
Rawmarsh
Rotherham
pound
nbsp
nbsp
bed
nbsp
nbsp
Fully
managed
high
yielding
investment
over
properties
under
management
nbsp
nbsp
nbsp
nbsp
Investment
Highlights
nbsp
nbsp
✓
RICS
valuation
✓
Full
title
from
day
✓
Professionally
planned
refurbished
✓
FREE
furniture
throughout
nbsp
nbsp
✓
Fully
tenanted
fully
managed
✓
HMO
compliancy
and
licenses
✓
Contracted
developer
Buy
Back
Option
✓
Hands
off
investment
nbsp
nbsp
nbsp
If
this
e-mail
is
not
displayed
properly
click
here
You
are
receiving
emails
from
us
as
your
email
is
part
of
business-to-business
email
database
which
was
created
from
email
addresses
available
in
the
public
domain
Should
you
wish
not
to
receive
any
further
communications
please
use
the
unsubscribe
link
on
the
email
If
you
want
to
unsubscribe
from
our
mailing
list
unsubscribe
here
nbsp
nbsp
nbsp
nbsp
If
you
want
to
unsubscribe
from
our
mailing
list
unsubscribe
here
nbsp
nbsp
nbsp
nbsp
-I9iGNUvPzu--
